src/components/WebPage.js

```diff
@@ -263,7 +263,9 @@ const WebPage = ({
 
             <ResourceTable value={about} schema={schema} />
 
+            { user &&
```

Comment:
Comments should be shown to anonymous user, too! Only adding a comment should be restricted to logged in users.